ascii_output_v3.py

```diff
@@ -0,0 +1,100 @@
+"""
+Turns csv output of tax calculator to ascii output with uniform columns
+and transposes data so columns are rows, rows are columns
+
```

Comment:
This description of the work makes me think that this should be done with `pandas` commands. Is there a strong reason to not consider DataFrame manipulations and then the `to_csv` method on DataFrames to do this work?